src/views/InformationCreation.jsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react'
+import { Redirect } from 'react-router-dom'
+import {
+  Box,
+  Typography,
+  TextField,
+  Button,
+  FormGroup,
+  FormControlLabel,
+  Checkbox,
+} from '@material-ui/core'
+import Information from '../modules/Information'
+import infoCreationTheme from '../theme/articleCreationTheme'
+import SuccessSnackbar from '../components/popups/SuccessSnackbar'
+import useCommonStyles from '../theme/useCommonStyles'
+
+const InformationCreation = () => {
+  const commonClasses = useCommonStyles()
+  const classes = infoCreationTheme()
+  const [redirect, setRedirect] = useState(false)
+  const [info, setInfo] = useState({
+    publish: true,
+    pinned: false,
+    header: '',
+    description: '',
+    link: '',
+  })
+  const descriptionMaxLength = 300
+
+  const handleChange = (event) => {
+    setInfo({
+      ...info,
+      [event.target.name]: event.target.value,
+    })
+  }
+
+  const handleSubmit = async (event) => {
+    event.preventDefault()
+    Information.create(info)
+    setRedirect(true)
+  }
+
+  return (
+    <Box className={commonClasses.viewContainer}>
+      <SuccessSnackbar />
+      <Box className={commonClasses.dashboardHeader}>
+        <Typography
+          data-cy='dashboard-header'
+          variant='h5'
+          style={{ fontWeight: 600 }}>
+          Create a new article
+        </Typography>
+      </Box>
+      {redirect && <Redirect to='/information' />}
+      <form
+        noValidate
+        autoComplete='off'
+        className={classes.formGroup}
+        data-cy='new-article-modal'
+        onSubmit={handleSubmit}>
+        <FormGroup row>
+          <FormControlLabel
+            control={
+              <Checkbox
+                data-cy='publish'
+                value={info.publish}
+                onChange={handleChange}
+                name='publish'
+                color='primary'
+              />
+            }
+            label='Publish'
+          />
+          <FormControlLabel
+            control={
+              <Checkbox
+                data-cy='pinned'
+                value={info.pinned}
+                onChange={handleChange}
+                name='pinned'
+                color='primary'
+              />
+            }
+            label='Pinned'
+          />
+        </FormGroup>
+        <TextField
+          className={classes.form}
+          data-cy='header'
+          variant='outlined'
+          fullWidth
+          required
+          id='standard-required'
+          label='Header'
+          type='string'
+          name='header'
+          onChange={handleChange}
+        />
```

Comment:
I usually also add attribute `value={state value that is changed onChange}` to the inputs. I am not sure if it is necessary here. Did you check that it sends correct object with all the values to the API?

Reply:
was not needed but had to to add a different on change handler (handleChecked)